mscore/timeline.cpp

```diff
@@ -926,8 +944,14 @@ void Timeline::drawGrid(int global_rows, int global_cols)
                                                                 grid_height * row + verticalScrollBar()->value(),
                                                                 getWidth(),
                                                                 grid_height);
-            meta_row->setBrush(QBrush(QColor(211,211,211)));
-            meta_row->setPen(QPen(QColor(150, 150, 150)));
+            if (preferences.isThemeDark()) {
+                  meta_row->setBrush(QBrush(QColor(75,75,75)));
+                  meta_row->setPen(QPen(QColor(50, 50, 50)));
+                  }
+            else {
+                  meta_row->setBrush(QBrush(QColor(211,211,211)));
+                  meta_row->setPen(QPen(QColor(150, 150, 150)));
```

Comment:
Better to do so:   

1. Add private structure   
```
struct Theme {
colors...
}
Theme _theme;
```   
2. Add method   
```
const Theme& theme() {
    return _theme;
}
```  

3. In the meyhod `updateTimelineTheme` set needed color   
4. Use theme anywhere 
```
meta_row->setBrush(QBrush(theme().metaBackgroundColor));
meta_row->setBrush(QPen(theme().metaTextColor));
```

This will give great benefits:
* All colors in one place, not scattered throughout the class.
* Simpler code in the class itself, no need to make these many ifs
* It’s easy to add another theme if suddenly we add support for themes

Reply:
I was thinking of having one main color for each theme and then offsets to automatically create the other colors (to automate theme changes) but this is better.